cylc/flow/task_events_mgr.py

```diff
@@ -1396,8 +1397,12 @@ def _process_message_succeeded(self, itask, event_time, forced):
             "run_status": 0,
             "time_run_exit": event_time,
         })
-        # Update mean elapsed time only on task succeeded.
-        if itask.summary['started_time'] is not None:
+        # Update mean elapsed time only on task succeeded,
+        # and only if task is running in live mode:
+        if (
+            itask.summary['started_time'] is not None
+            and itask.run_mode == RunMode.LIVE.value
+        ):
```

Comment:
Why?

I think this removes the task progress animation in the UI

Reply:
Legit point: The intention was to not have 0 values from skip mode pollute the mean time supplied to the GUI.